go.mod

```diff
@@ -15,6 +15,7 @@ require (
 
 require (
 	github.com/KyleBanks/depth v1.2.1 // indirect
+	github.com/cpuguy83/go-md2man/v2 v2.0.3 // indirect
```

Comment:
This seems out of place. Maybe it leaked in because of some other library?

Reply:
`go mod tidy` fixed this